File: Platron/Pages/ShopCartPage.cs
using System;
using OpenQA.Selenium;
using SeleniumExtras.PageObjects;

namespace Platron.Pages
{
    public class ShopCartPage : BasePage
    {
        public string RowsOnBinPage = "//tbody/tr[contains(@class, 'cart-form')]";
        public string RemoveRowOnBinPage = ".//preceding-sibling::td[@class='product-remove']";

        [FindsBy(How = How.XPath, Using = "//p[@class='cart-empty'][text()=.]")]
        public IWebElement TheBinIsClearMessage { get; set; }
    }
}

File: Platron/src/Pages/ShopCartPage.java
import org.openqa.selenium.By;

public class ShopCartPage extends  BasePage {
    public String RowsOnBinPage = "//tbody/tr[contains(@class, 'cart-form')]";
    public String RemoveRowOnBinPage = ".//preceding-sibling::td[@class='product-remove']";

    By TheBinIsClearMessage = By.xpath("//p[@class='cart-empty'][text()=.]");
}
